django_usaid_clinicfinder/clinicfinder/tasks.py

```diff
@@ -299,6 +285,56 @@ def run(self, lookuppointofinterest_id, **kwargs):
                  via Celery.',
                 exc_info=True)
 
+    def format_match_aat(self, match):
+        return "%s (%s)" % (
+            match.get('OrganisationName'),
+            match.get('FullAddress'))
+
+    def search_aat(self, lookuppoi):
+        url = (
+            "%(url)s?username=%(username)s&password=%(password)s&meters=50000"
+            "&category=%(category)s&x=%(x)s&y=%(y)s") % {
+                'url': settings.AAT_API_URL,
+                'username': settings.AAT_USERNAME,
+                'password': settings.AAT_PASSWORD,
+                'category': self.get_aat_category_id,
```

Comment:
We need to call `self.get_aat_category_id` -- we can't pass in the function itself to be formatted. We also need a test that checks this part.

You'll need something like `self.get_aat_category_id(lookuppoi.search)`.